recipes/napari-lazy-openslide/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "napari-lazy-openslide" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-lazy-openslide-{{ version }}.tar.gz
+  sha256: 4776c66e901a63c5e7afd55ada2278063b57c127cceceefab47063945abbd5f9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools-scm
+  run:
+    - dask-core >=2.20.0
+    - napari-plugin-engine >=0.1.4
+    - numpy >=0.1.19
```

Is this correct? I'd assume it means:

```suggestion
    - numpy >=1.19
```